scripts/main/lychee.js

```diff
@@ -213,24 +226,45 @@ lychee.logs = function () {
  * @returns {void}
  */
 lychee.aboutDialog = function () {
-	const msg = lychee.html`
-				<h1>Lychee ${lychee.version}</h1>
-				<div class='version'><span><a target='_blank' href='${lychee.updateURL}'>${lychee.locale["UPDATE_AVAILABLE"]}</a></span></div>
-				<h1>${lychee.locale["ABOUT_SUBTITLE"]}</h1>
-				<p>${sprintf(lychee.locale["ABOUT_DESCRIPTION"], lychee.website)}</p>
-			  `;
+	const aboutDialogBody = `
+		<h1>Lychee <span class="version-number"></span></h1>
+		<p class="update-status up-to-date"><a target='_blank' href='${lychee.updateURL}'></a></p>
```

Comment:
Do we need this `up-to-date` here, considering that we programmatically add it as-needed in `initAboutDialog` below?

Reply:
I added it, to theoretically avoid flicker in short time between the dialog has been opened and before the method `initAboutDialog` has been completely executed. If the  class `up-to_date` is not defined on the paragraph, the paraph is initially visible and then vanished if the class is added later programmatically. However, I say "theoretically", because I never experienced it in practice. Probably, modern systems are simply fast enough.

Yes, you a right, in the code

```php
if (lychee.update_available) {
	updClassList.remove("up-to-date");
} else {
	updClassList.add("up-to-date");
}
```

below, I could remove the `else`-branch, because `"up-to-date"` there is no point in adding it. Shall I?

An an optimal world, I world we would change `basicModal` as follows: first construct the the dialog on the shadow DOM (we do that), then call the provided initializing callback and when the callback has completed mount the shadow DOM into the actual DOM. This would avoid any flicker under all circumstances. (Also for more complex dialogs.) It would ensure that the whole dialog becomes only visible at once after it has been fully initialized. May be I do that.

Reply:
I removed the unnecessary `else`-branch in https://github.com/LycheeOrg/Lychee-front/commit/8279248d69de6ec8e4a7df7f597cf302c25e3cb2.